File: dOC/CoreDataDemo/CoreDataDemo/Person+CoreDataProperties.m
//
//  Person+CoreDataProperties.m
//  CoreDataDemo
//
//  Created by runo on 17/6/16.
//  Copyright © 2017年 com.runo. All rights reserved.
//

#import "Person+CoreDataProperties.h"

@implementation Person (CoreDataProperties)

+ (NSFetchRequest<Person *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Person"];
}

@dynamic age;
@dynamic name;
@dynamic dogMaster;

@end

File: CoreDataDemo/CoreDataDemo/Person+CoreDataProperties.swift
//
//  Person+CoreDataProperties.swift
//  CoreDataDemo
//
//  Created by runo on 17/1/4.
//  Copyright © 2017年 com.runo. All rights reserved.
//

import Foundation
import CoreData


extension Person {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Person> {
        return NSFetchRequest<Person>(entityName: "Person");
    }

    @NSManaged public var name: String?
    @NSManaged public var age: Int16
    @NSManaged public var gendar: String?
    @NSManaged public var card: Card?

}
